.evergreen/setup-env.sh

```diff
@@ -1,8 +1,17 @@
 set -e
 set -x
 
+OS_ARCH="$(uname "-m")"
+if [ "$OS_ARCH" = "aarch64" ] || [ "$OS_ARCH" = "arm64" ] ; then
+    echo "[INFO] Choosing v4 because OS_ARCH is $OS_ARCH"
+    export TOOLCHAIN_PATH='/opt/mongodbtoolchain/v4/bin'
```

Comment:
The idea here is that we'd pick v4 on ppc64le and s390x – not sure what the `uname -m` values are for that, but we should probably be using those?

Reply:
Yeah, I was absentmindedly reverting that commit and got distracted before I changed the values 🤦‍♂️ One thing though - do we need to pick v4 for s390x builds - those are already passing successfully with the v3 toolchain.

Reply:
Yeah, if v3 works for s390x, then it probably doesn't matter for now 🤷‍♀️